src/data/faq.json

```
@@ -45,6 +45,16 @@
                             "<b>National:</b> 0800 7540002",
                             "<b>International:</b> +49 30 498 75402"
                         ]
+                    },
+                    {
+                        "title": "How do I create an error log?",
+                        "anchor": "error_log",
+                        "active": true,
+                        "textblock": [
+                            "With version 2.2, the Corona-Warn-App offers a new function for detailed technical problem analysis. The error log is intended to help the developers to faster perform a root cause analysis and to eliminate errors more efficiently.<br><b> Important:</b> Please report an issue <b>first</b> to the hotline (national: 0800 7540001 or international: +49 30 498 75401) to verify if there is already a known solution. An error log is only required if the CWA team asks for this report.",
+                            "<ol><li>If so, please open a ticket on GitHub and describe the problem behavior as detailed as possible.</li><li>Start logging in the CWA (---LINK or screenshot here: How do I start the logging?---).<br>-Try to reproduce the behavior or try generate the error message.<br>- If this attempt was successful, send the technical log to the CWA server. The data will be anonymized and encrypted. (If you are interested, you may also save the report locally on your Smartphone.)<br>- In return you will receive a unique ID, which you can also find in the list of all IDs.<br>- Stop the error log.</li><li>Update the GitHub ticket with your unique identifier from the server so that the report can be linked to your ticket.</li></ol>",
+                            "Remarks:<ul><li>Technical logs are automatically deleted after 14 days.</li><li>Logs that are transmitted to the server without request by the CWA team cannot be analyzed and processed.</li><li>There is no back channel into the Corona Warning app for the error logs. The CWA technical team can only communicate via GitHub to a ticket.</li></ul>"

```

Align text with app:

```suggestion
                            "Remarks:<ul><li>Error reports are automatically deleted after 14 days.</li><li>Logs that are transmitted to the server without request by the CWA team cannot be analyzed and processed.</li><li>There is no back channel into the Corona Warning app for the error report. The CWA technical team can only communicate via GitHub to a ticket.</li></ul>"
```

---

same ... thanks!